### PR TITLE
feat: add importer for Digital Credit Union

### PR DESCRIPTION
**Pull request by hlieberman**
This adds a CSV importer for Digital Federal Credit Union (DCU), the 18th largest credit union in the United States (by total assets under management). Unfortunately, DCU doesn't offer OFX exports, so this is the best we can do.

### CODE REVIEW

File: beancount_reds_importers/importers/dcu/__init__.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+"""DCU (Digital Federal Credit Union) ofx importer for beancount."""
```
Comment (redstreet) [inline beancount_reds_importers/importers/dcu/__init__.py:1]: ofx -> csv

Comment (hlieberman) [reply beancount_reds_importers/importers/dcu/__init__.py:1]: Derp.  One sec, pushing a fix now.

**Comment by redstreet** [comment]
Nice, thanks! For csv importers, it's far more important to have unit tests (compared to ofx), so would you mind adding at least a couple lines? It's simple: `beancount_reds_importers/importers/schwab/tests/schwab_csv_checking` is a good example.

**Comment by hlieberman** [comment]
Yeah, no problem.  Would you mind if I pulled in another dependency, at least as dev dep? (I try, whenever possible, to add hypothesis tests if I'm going to write unit tests -- in addition to a static test.)

**Comment by redstreet** [comment]
Sure, dev dependencies are fine.

**Comment by hlieberman** [comment]
@redstreet, I ended up not using hypothesis after all.  I wrote up a test, but it really is testing beancount more than it's testing the importer, since it's making sure a series of random transactions that balance before import, balance after import.

**Comment by redstreet** [comment]
Makes sense, and sounds good. If you don't mind addressing the comment above about `'USD'`, I can merge this in.

**Comment by hlieberman** [comment]
> the comment above about `'USD'`

I'm sorry, I must be blind; I don't see anything about USD above.  The only comment I see on the code is the ofx => csv one.

**Review by redstreet — CHANGES_REQUESTED** [review]

**Comment by redstreet** [comment]
> I'm sorry, I must be blind; I don't see anything about USD above.  The only comment I see on the code is the ofx => csv one.

Drat, completely my bad, sorry! Github requires separately hitting a "submit review" as a second step when adding a comment as a part of a review (as opposed to adding a freestanding comment). This is non-obvious and not at all the first time I've been tripped up by it. It should be visible now.

File: beancount_reds_importers/importers/dcu/__init__.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+                "CURRENT BALANCE": "balance",
+                "TRANSACTION TYPE": "type",
+            }
+
+            self.transaction_type_map = {
+                "DEBIT": "transfer",
+                "CREDIT": "transfer",
+            }
+            self.skip_transaction_types = []
+            self.currency = "USD"
```
[the remark this answers is not in the document]
Comment (hlieberman) [reply beancount_reds_importers/importers/dcu/__init__.py]: No problem; change pushed.

**Comment by hlieberman** [comment]
> Drat, completely my bad, sorry! Github requires separately hitting a "submit review" as a second step when adding a comment as a part of a review (as opposed to adding a freestanding comment).
Ah yes, I've been bitten by this one myself a couple of times.  Change made!

**Comment by redstreet** [comment]
Awesome, thank you!

**Comment by redstreet** [comment]
MErged

**Pull request merged**